src/NATS.Client.JetStream/NatsJSContext.Streams.cs

```diff
@@ -17,6 +17,7 @@ public partial class NatsJSContext
         StreamConfig config,
         CancellationToken cancellationToken = default)
     {
+        ArgumentNullException.ThrowIfNull(nameof(config.Name));
```

Comment:
missing argument.

Reply:
Good catch. I had two other places with the same mistake! .. at least I'm consistently wrong ;)